packages/core/forms/src/directives.ts

```diff
@@ -0,0 +1,31 @@
+import type { ObjectDirective, DirectiveBinding, VNode } from 'vue'
+import {
+  get as objGet,
+  forEach,
+  isString,
+} from 'lodash'
+
+const bindFieldTypeAttrs = (el: any, binding: DirectiveBinding<any>, vnode: VNode<any, any, { [key: string]: any }>) => {
+  const container = binding.value || 'input'
+  let attrs = objGet(vnode.props, 'schema.attributes', {})
+
+  if (isString(container)) {
+    attrs = objGet(attrs, container) || attrs
+  }
+
+  forEach(attrs, (val, key) => {
+    el.setAttribute(key, val)
+  })
+}
+
+const attributesDirective: ObjectDirective = {
+  beforeMount(el, binding, vnode) {
+    bindFieldTypeAttrs(el, binding, vnode)
+  },
+  updated(el, binding, vnode) {
+    bindFieldTypeAttrs(el, binding, vnode)
+  },
+  // Vue 3 does not have a "componentUpdated" lifecycle hook equivalent
+}
```

Comment:
I tried to find external usages of `v-attribtues` throughout `entities-plugins` and `konnect-ui-apps`, but I only found the following usage:

https://github.com/Kong/public-ui-components/blob/4a9f171020980adf3ead078b4a34fff121044768/packages/entities/entities-plugins/src/composables/useSchemas.ts#L577-L586

It reminds me that we also have `inputAttributes` in our VFG schema, and I wonder if they overlap: 🤔

https://github.com/Kong/public-ui-components/blob/4a9f171020980adf3ead078b4a34fff121044768/packages/entities/entities-plugins/src/definitions/schemas/typedefs.ts#L48-L66

Reply:
I think I might have been wrong about that overlap. I suggest let's try taking them out. It looks like we may not use it anyway.